Src/Newtonsoft.Json-for-Unity/CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Newtonsoft.Json for Unity changelog
 
-## 12.0.1 _`(patch 001)`_
+## 12.0.101
+
+- Changed versioning. For more info see [the readme in the repository][repo-url]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4554595)